### PR TITLE
Redesign About page, add License and Source links

### LINKED ISSUE #12: [Usability issues] Minor usability flaws in Patchmanager's "About" dialogue
1. Patchmanager's "About" dialogue does not reveal the version number of the installed Patchmanager.

1. One cannot scroll in Patchmanager's "About" dialogue to its bottom in landscape orientation. 

Tested with Patchmanager 2.3.1-10.39.1.jolla on a Jolla 1 phone under SFOS 2.1.0.11

Originally reported [on TMO as issue 3.](http://talk.maemo.org/showpost.php?p=1527185&postcount=2208) (due to issue #9; filing this here [as promised](http://talk.maemo.org/showpost.php?p=1529055&postcount=2245)).

### PR DESCRIPTION
**Pull request by nephros**
NOTE: license given is BSD-3 for now, clarification still open about that, see in PR #61

Fixes #12

https://github.com/sailfishos-patches/patchmanager/issues/12

### CODE REVIEW

**Review by b100dian — COMMENTED** [review]
Tested, looks good -see the nits for translation args.
But are we going to live the commented code in? It's in the history.
Also does it fix the scrolling? I also see a tiny bit of scrolling on my landscape phone, I assume the same holds for landscape tablet?

File: src/qml/AboutPage.qml [hunk 1]
```diff
@@ -81,27 +81,93 @@ Page {
...
+
+                Separator {
+                        width: parent.width
+                        color: Theme.primaryColor
+                        horizontalAlignment: Qt.AlignHCenter
+                }
+
+                Text {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        text: qsTranslate("", "Credits and Thanks<br />")
```
[the remark this answers is not in the document]
Comment (nephros) [reply src/qml/AboutPage.qml:125]: I agree, but right now qsTranslate is *all over* the codebase.

We can do a janitorial move to qsTr() in a big cleanup task but right now I'd like to be consistent.

**Comment by nephros** [comment]
> But are we going to live the commented code in? It's in the history. 

Will rip commented code in a fixup.

> Also does it fix the scrolling? I also see a tiny bit of scrolling on my landscape phone, I assume the same holds for landscape tablet?

Can you give some details? I'm not aware of problems with scrolling.

File: src/qml/AboutPage.qml [hunk 2]
```diff
@@ -81,27 +81,93 @@ Page {
...
-                    anchors.margins: Theme.horizontalPageMargin
-                    anchors.verticalCenter: parent.verticalCenter
-                    text: qsTranslate("", "Developers")
+            Column {
+                width: parent.width - Theme.paddingSmall
+                spacing: Theme.paddingLarge
+
+                Text {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        text: qsTranslate("", "Licensed under the terms of the<br />")
```
[the remark this answers is not in the document]
Comment (nephros) [reply src/qml/AboutPage.qml]: done!

**Comment by b100dian** [comment]
> Can you give some details? I'm not aware of problems with scrolling.

In the original issue Olf says that scrolling in landscape (tablet, probably) only scrolls a bit. I've also noticed the 'scrolls a bit' problem on portrait/phone - meaning there's something wrong with the outer elements?

For example, removing `contentHeight: mainColumn.height` from SilicaFlickable does stop that tiny scrolling, but I don't know if it fixes (adds correct scrolling) in landscape / on the tablet (because the default-allowed-orientations does not work on 64 bits, right?:p actually I need to modify the settings application to allow landscape)

**Comment by b100dian** [comment]
> For example, removing contentHeight: mainColumn.height from SilicaFlickable does stop that tiny scrolling

It's not that, i just changed settings.qml to stop setting allowed orientations. It's something else, maybe the easter code:)

**Comment by b100dian** [comment]
Yup, this guy 
```
            Item {
                width: 1
                height: flick.height - y
            }
```

**Comment by nephros** [comment]
Nice find - I for one will not touch the easter code though.

I could do with a revamping - the effect doesn't really look very good on modern high-res displays.

**Review by b100dian — APPROVED** [review]
Let's get this in as it is - I am not sure if removing that Item breaks Easter mode or not, it seems broken-ish anyway. We can make a separate PR for @Olf0 to test the scrolling without that item:D

**Comment by Olf0** [comment]
> Let's get this in as it is - I am not sure if removing that Item breaks Easter mode or not, it seems broken-ish anyway.

I fail to trigger the easter-egg, but do see a white line (as if a scroll-bar) on the right side of the "About"-page.  This is missing on the "Developer"-subpage.

> We can make a separate PR for @Olf0 to test the scrolling without that item:D

The easter-egg?  Seems hard to trigger, with its width=1; I fail to with PM3.0.65.
What does it do?  Well, per definition of an easter-egg, it is not necessary, and if it is additionally broken, throwing it out might make sense.
 
BTW, on an XperiaX@SFOS3.2.1 with PM3.0.65 scrolling of the "About"-page basically works in landscape orientation, but is not fluid (it feels strangely choppy)!  In contrast to the the "Developer"-subpage, where scrolling works as usual.

**Pull request merged**